docker/Dockerfile

```diff
@@ -31,7 +30,7 @@ ENV BK_USER=bookkeeper
 # Download Apache Bookkeeper, untar and clean up
 RUN set -x \
     && adduser "${BK_USER}" \
-    && yum install -y java-1.8.0-openjdk-headless wget bash python md5sum sha1sum sudo \
```

Comment:
any reason removing `md5sum` and `sha1sum`? I think we need these two commands. even if these two commands are pre-packaged in the base image, there is no harm to install these two commands, right? It actually make the dependencies clear, no?